napari/utils/transforms/transforms.py

```diff
@@ -119,6 +121,12 @@ def inverse(self) -> 'TransformChain':
         """Return the inverse transform chain."""
         return TransformChain([tf.inverse for tf in self[::-1]])
 
+    @property
+    def is_diagonal(self):
+        if all(getattr(tf, 'is_diagonal', False) for tf in self):
```

Comment:
Instead of using a default of `False` for `getattr`, what do you think about defining `is_diagonal` in the base `Transform` class and having it return `False`?